src/satosa/attribute_mapping.py

```diff
@@ -145,8 +145,11 @@ def _handle_template_attributes(self, attribute_profile, internal_dict):
 
         return internal_dict
 
-    def _get_nested_attribute_value(self, nested_key, data):
-        keys = nested_key.split(self.separator)
+    def _get_nested_attribute_value(self, nested_key, data, profile):
+        if (profile == 'openid'):
```

Comment:
`profile` (or `attribute_profile`) is a _configuration value_ – it could be anything. I cannot be checked like that.

Reply:
Well, as far as I understand 'openid' mapping key is fixed for all oidc frontends and backends as they appear in current codebase? See e.g. frontends/openid_connect.py @125
```
    attributes = self.converter.from_internal("openid", internal_resp.attributes)
```
All references to internal_attributes mapping dict refer to 'openid' as their mapping? It was my intention to further rely on that convention?

Reply:
Well, that is bad. I see it hardcoded in other places too. This means that there cannot be an OIDtoOID proxy (as one would do a SAMLtoSAML proxy) – or in other words, the behaviour for openid is special. 

The `attribute_profile` should be stored at `self.attribute_profile`, and retrieved from there. I will look more into it.

Reply:
Hi guys,

What's the best way going forward?  Might I suggested accepting this pull request (unless there are other pressing issues that we need to fix), and after that fixing the OpenID code path to remove hardcoded values?